checker/tests/nullness/EnsuresNonNullIfTest2.java

```diff
@@ -8,6 +8,7 @@
 // Test case for a mysterious error.
 abstract class EnsuresNonNullIfTest2a {
     @EnsuresNonNullIf(result=true, expression="xxx")
+    //:: error: (flowexpr.parse.error)
```

Comment:
Wouldn't it be nicer if the error were issued on the `@EnsuresNonNullIf`, not on the method declaration? There might be quite a distance between the two.